crates/biome_cli/tests/commands/format.rs

```diff
@@ -1281,7 +1280,10 @@ fn does_not_format_ignored_files() {
     let mut console = BufferConsole::default();
     let mut fs = MemoryFileSystem::default();
     let file_path = Utf8Path::new("biome.json");
-    fs.insert(file_path.into(), CONFIG_FORMATTER_IGNORED_FILES.as_bytes());
+    fs.insert(
```

Comment:
q: are we no longer using the constant here to improve maintainability (switching from an ignore-based configuration to an includes-based one with a negation)?

Reply:
I am not sure to understand what you mean.

This function is a test, I chose to use an inline configuration because this makes it easier to understand and edit the test.
This makes the test self-contained.

Reply:
I am going to merge the PR. Please let me know if I missed something. I will fix it in a future PR if needed.